config/optional/block.block.umami_banner_recipes.yml

```diff
@@ -1,21 +1,31 @@
+uuid: null
 langcode: en
 status: true
 dependencies:
+  enforced:
```

Comment:
@larowlan This makes the configuration dependent on **demo_umami_content** module and uninstallation, re-installation leads to removal of this block.